setup.py

```diff
@@ -20,5 +20,5 @@
         "Topic :: Software Development :: Quality Assurance",
         "Topic :: Software Development :: Testing",
     ],
-    install_requires=["coverage", "pygount"],
+    install_requires=["coverage==5.3", "pygount==1.2.4"],
```

Comment:
Nobody will know that coverage2clover fails with new versions of required libraries :) Usually I add a requirements.txt and enable [PyUp](https://pyup.io) to receive PRs with updated dependencies. 
I don't know which way is better :(

Reply:
Correct. Even I am not sure which way is better. Usually, a practice I hope which is more common is that people would be freezing to `coverage2clover==2.0.1`  or some v2. Updating to v3 denotes clearly that the repository might be introducing breaking changes. But this I will leave for you to decide about.

I am not familiar with PyUp, so can't really comment on that, sorry 😅

Reply:
In fact, I tried to install `coverage2clover`  and this is what I get

```
Collecting coverage
  Using cached coverage-5.3-cp38-cp38-macosx_10_9_x86_64.whl (205 kB)
Collecting pygount
  Using cached pygount-1.2.4-py3-none-any.whl (25 kB)
Collecting pygments>=2.0
  Using cached Pygments-2.7.0-py3-none-any.whl (950 kB)
Installing collected packages: coverage, pygments, pygount, coverage2clover
Successfully installed coverage-5.3 coverage2clover-2.0.1 pygments-2.7.0 pygount-1.2.4
```

So I see that v2.0.1 installs `coverage-5.3` internally. But the unit tests (in the older 2.0.1) suggest the test being run against 4.3.4, which is kind of misleading, because any user of the tool today are already using coverage 5.3 behind the scenes probably.

Reply:
"open" version in setup.py allows to skip some maintenance while library still works with recent versions of dependencies, but there should be a way to automatically ensure that library is not broken _before_ users open issues in gh. So, setup.py still has "open" versions (if you want, you may downgrade `coverage` etc in your CI pipeline), and requirements.txt points to latest release.